src/utils/menu.js

```diff
@@ -1,52 +1,75 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Projects',
```

Comment:
Lets use "1Hive" if it's at the start of a sentence, as much as others might disagree I think 1hive looks unprofessional, especially amongst other titles with proper capitals.

Reply:
okay I agree, I reluctantly went with 1hive because most other places had the 'h' lowercase. I wasn't really sure why but I went with it.